pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -29,7 +38,67 @@
 PULP_SERVICES = ("pulpcore-content", "pulpcore-api", "pulpcore-worker@1", "pulpcore-worker@2")
 
 
+## Pulp No Leftover Objs Feature
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup("pulp-smash")
+    group.addoption(
+        "--pulp-no-leftover-objs",
+        action="store_true",
+        dest="pulp_no_leftover_objs",
+        help="Enable this to have Pulp plugins check for objects leftover by tests.",
```

Comment:
```suggestion
        "--pulp-no-leftovers",
        action="store_true",
        dest="pulp_no_leftovers",
        help="Enable this to have Pulp plugins check for objects leftover by tests.",
```

I thought calling them objects is redundant. And saving few keystrokes will hopefully help people to run it more often.

Reply:
I changed this in my next push.